FAQ: Link to gcore blog post (#6)
canonical/dqlite-docs: explanation/faq.md

* No need for statements to be deterministic (e.g. you can use `time()` )
* Frame-based replication instead of statement-based replication

> [Here](https://gcore.com/blog/comparing-litestream-rqlite-dqlite/) is a blog post comparing dqlite with rqlite and Litestream (another implementation of replication for SQLite).

Reviewer: "Here" is a bad link title for SEO (both for the site your linking to and for our docs).
Maybe:
```suggestion
See the [Comparing Litestream, rqlite, and dqlite](https://gcore.com/blog/comparing-litestream-rqlite-dqlite/) blog post for a comparison of different SQLite replication implementations.
```
